spanner/ddl/ast.go

```diff
@@ -292,6 +292,9 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 		}
 	}
 
+	if len(keys) == 0 {
+		return fmt.Sprintf("%sCREATE TABLE %s (\n%s) %s", tableComment, config.quote(ct.Name), cols, interleave)
```

Comment:
getting syntax error due to extra `,` : Error parsing Spanner DDL statement: CREATE TABLE `test123` ( `name` STRING(32), `age` INT64, ) : Syntax error on line 4, column 2: Expecting 'PRIMARY' but found 'EOF'